cluster/helm-operator/deployment.yaml

```diff
@@ -0,0 +1,160 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: helm-operator
+  namespace: flux-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: helm-operator
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        name: helm-operator
+      annotations:
+        prometheus.io/scrape: "true"
+    spec:
+      serviceAccountName: helm-operator
+      volumes:
+      #
+      # You will need these two volumes if you want to establish validated TLS
+      # connections against Tiller.
+      #
+      # - name: tiller-tls-ca
+      #   configMap:
+      #     name: tiller-tls-ca-config
+      #     defaultMode: 0600
+      #
+      # Secret of type `kubernetes.io/tls`:
+      #
+      # - name: tiller-tls-cert
+      #   secret:
+      #     secretName: tiller-tls-cert
+      #     defaultMode: 0400
+      #
+      # The following volume is for using a customised `known_hosts` file,
+      # which you will need to do if you host your own git repository rather
+      # than using GitHub or the like. You will also need to mount it
+      # into the container, below.
+      #
+      # You may also wish to provide an `ssh_config` file,
+      # mentioning more than one `IdentityFile`; for instance, if you are
+      # using more than one GitHub repository. NB: the entry key should be
+      # "config" rather than "ssh_config" if mounted in ~/.ssh/.
+      #
+      # - name: sshdir
+      #   configMap:
+      #     name: flux-ssh-config
+      #     defaultMode: 0400
```

Comment:
Do we need to enable the ssh configs here? Or could it be overridden separately for another namespace if the other namespace references our internal git?

Reply:
I think this config is more so that this repo can also auto-update itself with the repo, but since right now it exists in public git and not our bitbucket, we don't need it